src/main/java/com/crud/tasks/service/MailCreatorService.java

```diff
@@ -22,6 +26,8 @@ public String buildTrelloCardEmail(String message) {
         context.setVariable("tasks_url", "http://cezaryjanicki.github.io");
         context.setVariable("button", "Visit website");
         context.setVariable("admin_name", adminConfig.getAdminName());
+        context.setVariable("gooodbye_message", "Thank you for using our service. Have a nice day!");
+        context.setVariable("company", companyConfig.getCompanyName() + " " + companyConfig.getCompanyPhone());
```

Comment:
Raczej po to korzystamy z silnika szablonów, żeby nie składać frontendu na backendzie ;) Możesz tutaj spokojnie ustawić 2 zmienne w kontekście i wstawić je niezależnie w szablonie.